src/emqx_ctl.erl

```diff
@@ -232,3 +232,11 @@ zip_cmd([X | Xs], []) -> [{X, ""} | zip_cmd(Xs, [])];
 zip_cmd([], [Y | Ys]) -> [{"", Y} | zip_cmd([], Ys)];
 zip_cmd([], []) -> [].
 
+replace(Str0) ->
+    case re:run(Str0, "~n$") of
```

Comment:
is it to avoid line feed ?
then it should replaced `\n` instead (`\r` too).

Reply:
I need to replace `~` with `~~`, but at the same time avoid replacing the end of the line with `~n`,